form_style.css

```diff
@@ -0,0 +1,63 @@
+h2{
```

Comment:
It's great to have consistent formatting for our code. A linter like Prettier (ext you can get on BSCode) will do the formatting for us on save.

Example of best practice CSS formatting: 

```
h2 {
    color: ghostwhite;
    text-align: center;
    font-size: 50px;
}
```